2_run.R

```diff
@@ -17,6 +17,18 @@ p2 <- list(
         model_config = p2_nldas_model_config_filt,
         calibrate = FALSE)
     },
-    pattern = map(p2_nldas_model_config_filt)
+    pattern = map(p2_nldas_model_config_filt),
+    format = 'file'
+    ),
+
+  tar_target(
+    p2_nldas_glm_uncal_tibble,
```

Comment:
SC: If you wanted, you could modify these tibbles to include a bit more info (e.g., site_id, simulation directory, model years, driver_start_date, driver_end_date), which would then mean that you don't have to pull that info out in `extra_glm_output()`. Might be worth considering if that information would be handy to use elsewhere, e.g., when plotting.